fb/usage.go

```diff
@@ -50,10 +51,22 @@ type NameID struct {
 	Name string `json:"name"`
 }
 
-func (fbClient FlashbladeClient) Usage() (UsageResponse, error) {
+func FilterFilesystems(vs []FilesystemsItem, regexMatch string) []FilesystemsItem {
```

Comment:
This function is public, but it could be private as it's only used internally to the fb package.